pkg/services/sqlstore/annotation_cleanup_test.go

```diff
@@ -25,13 +27,15 @@ func TestAnnotationCleanUp(t *testing.T) {
 
 	createTestAnnotations(t, fakeSQL, 21, 6)
 	assertAnnotationCount(t, fakeSQL, "", 21)
+	assertAnnotationTagCount(t, fakeSQL, 42)
```

Comment:
I agree that it hard to test has a test has the expected behaviour by checking the count.
The best alternative I could think would be to check that each time the tags should equal the summary of the remaining annotations multiplied by two (since each created annotation has two tags)

Reply:
OK, I will include some logic so assert against the sum of the annotation counts * 2, instead of having it explicitly written  as part of the test case.